release-notes/unreleased/2949-coarse-location-android.yml

```
@@ -0,0 +1,6 @@
+issue_key: 2949
+show_in_stores: true
+platforms:
+  - android
+en: A coarse location permission is now seen as an accepted permission
+de: Erlaubnis zur groben Ortung zählt jetzt als erlaubt
```

:wrench: Since release notes are visible to users, I think we should focus a little more on actual changes for them (and not what changed internally):
```suggestion
en: Fix location detection if only granted approximate permission.
de: Die Standortbestimmung funktioniert nun auch, wenn nur eine grobe Bestimmung erlaubt wird.
```